infra-templates/l2-flat/config.yml

```diff
@@ -0,0 +1,8 @@
+name: Rancher Layer2 Flat
+version: v0.1.0
+description: |
+  Rancher Networking plugin using Layer2 Flat.
+category: Networking
+labels:
+  io.rancher.certified: rancher
```

Comment:
can we remove this label so that the `Rancher Labs` banner is not on the catalog item?

Reply:
@deniseschannon Done